### ISSUE TITLE
Javascript error from Designer

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#12622]
### Steps to reproduce
1. Click on the Designer tab
2. (if needed) Press the "Save" link

### Expected behavior
Designer should load

### Actual behavior
The Javascript error is shown:
```
Notice in ./libraries/pmd_common.php#468
 Undefined index: t_h

Backtrace

./db_designer.php#48: PMA_saveTablePositions(string '4')
Warning in ./libraries/pmd_common.php#468
 Invalid argument supplied for foreach()

Backtrace

./db_designer.php#48: PMA_saveTablePositions(string '4')
```
### Server configuration
**Operating system**:
Debian
**Web server:**
Demo server
**Database:**
MySQL 5.5
**PHP version:**
5.6.24
**phpMyAdmin version:**
STABLE and master

### Client configuration
**Browser:**
Safari and Chrome
**Operating system:**
Mac OS

**Comment by devenbansod** [comment]
I am not able to exactly reproduce this but I get the same error in a slightly different scenario (may be they are unrelated too).

I go to Designer. It loads. A new page (Untitled*) is loaded. I click 'Save Page' and it asks to enter a name for the page.

Now if I click the 'Go' button, it gets submitted properly, but if I press Enter (expecting to submit this form), it gives the same error as the one you mentioned. 
Though this error should be fixed with [this](https://github.com/devenbansod/phpmyadmin/commit/dc5c7b0c6764261839940ea7cade2f0f2b006075.patch) patch (not yet merged), I am not sure if it fixes the error you are getting.

**Comment by nijel** [comment]
This error is quite frequent on the error reporting server...

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#28282](https://reports.phpmyadmin.net/reports/view/28282)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#27724](https://reports.phpmyadmin.net/reports/view/27724)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#27529](https://reports.phpmyadmin.net/reports/view/27529)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#27377](https://reports.phpmyadmin.net/reports/view/27377)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#26910](https://reports.phpmyadmin.net/reports/view/26910)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#26900](https://reports.phpmyadmin.net/reports/view/26900)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#26850](https://reports.phpmyadmin.net/reports/view/26850)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#26657](https://reports.phpmyadmin.net/reports/view/26657)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#26412](https://reports.phpmyadmin.net/reports/view/26412)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#26111](https://reports.phpmyadmin.net/reports/view/26111)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#25160](https://reports.phpmyadmin.net/reports/view/25160)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#24934](https://reports.phpmyadmin.net/reports/view/24934)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#24886](https://reports.phpmyadmin.net/reports/view/24886)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#24748](https://reports.phpmyadmin.net/reports/view/24748)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#24438](https://reports.phpmyadmin.net/reports/view/24438)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#24347](https://reports.phpmyadmin.net/reports/view/24347)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#24177](https://reports.phpmyadmin.net/reports/view/24177)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#23934](https://reports.phpmyadmin.net/reports/view/23934)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#20770](https://reports.phpmyadmin.net/reports/view/20770)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#21271](https://reports.phpmyadmin.net/reports/view/21271)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by nijel** [comment]
Param | Value 
 -----------|--------------------
 Error Type | Notice
 Error Message |Undefined index: t_h
 Exception Type |php
 Link | [Report#20826](https://reports.phpmyadmin.net/reports/view/20826)

*This comment is posted automatically by phpMyAdmin's [error-reporting-server](http://reports.phpmyadmin.net).*

**Comment by devenbansod** [comment]
Should be fixed with merging of PR #12648
